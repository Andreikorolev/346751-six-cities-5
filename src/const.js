export const Cities = [
  `Paris`,
  `Cologne`,
  `Brussels`,
  `Amsterdam`,
  `Hamburg`,
  `Dusseldorf`,
];

export const SortOptions = {
  popular: `Popular`,
  lowToHigh: `low to high`,
  highToLow: `high to low`,
  rating: `Top rated first`,
};
