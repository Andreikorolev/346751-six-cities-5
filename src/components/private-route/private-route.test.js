import React from "react";
import renderer from "react-test-renderer";
import {PrivateRoute} from "./private-route";
import {BrowserRouter} from "react-router-dom";
import {AuthorizationStatus} from "../../const";

describe(`PrivateRoute render`, () => {
  it(`should render with NO_AUTH`, () => {
    const tree = renderer
      .create(
          <BrowserRouter>
            <PrivateRoute
              exact={true}
              path={`path`}
              render={() => {}}
              authorizationStatus={AuthorizationStatus.NO_AUTH}
            />
          </BrowserRouter>
      ).toJSON();

    expect(tree).toMatchSnapshot();
  });

  it(`should render with AUTH`, () => {
    const tree = renderer
      .create(
          <BrowserRouter>
            <PrivateRoute
              exact={true}
              path={`path`}
              render={() => {}}
              authorizationStatus={AuthorizationStatus.AUTH}
            />
          </BrowserRouter>
      ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
