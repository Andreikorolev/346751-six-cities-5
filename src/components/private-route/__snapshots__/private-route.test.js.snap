// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PrivateRoute render should render with AUTH 1`] = `null`;

exports[`PrivateRoute render should render with NO_AUTH 1`] = `null`;
